static/app/views/issueDetails/streamline/sidebar/seerNotices.tsx

```diff
@@ -153,10 +153,7 @@ export function SeerNotices({groupId, hasGithubIntegration, project}: SeerNotice
           </Fragment>
         )}
         {nonGithubRepos.length > 0 && (
-          <Fragment>
-            {' '}
```

Comment:
Was this space here doing something?

Reply:
https://github.com/getsentry/sentry/pull/91642/commits/bc89939bd55e492855d2a8c1c4ae9f0213a7a7c2 looks like the automated linter is doing it

Reply:
ok that's fine